src/components/icon/GraphItemIcon/GraphItemIcon.module.css

```diff
@@ -4,5 +4,6 @@
 }
 
 .graphItemtIconTypeDiv span {
```

Comment:
Styling the AppIcon or any inner Icon component via a wrapper class and child element `span` is potential for regression. If AppIcon or Icon changes the HTML element from `span` some other markup tag, the CSS rules will no longer apply.

Reply:
It does appear these CSS rules will still apply to `Icon` component but uncertain of its computed styles.

https://github.com/folio-org/stripes-components/blob/master/lib/Icon/Icon.js#L96

Reply:
And here is the stripes-core `AppIcon` component documentation. https://github.com/folio-org/stripes-core/tree/master/src/components/AppIcon